locales/en/actions.json

```diff
@@ -22,5 +22,6 @@
   "print_move_list": "Print moves",
   "print_move_detail": "Print move",
   "sign_out": "Sign out",
-  "switch_location": "Switch location"
+  "switch_location": "Switch location",
+  "save": "Save and continue"
```

Comment:
Should this be `save_and_continue`? It's more explicit to the actual content that way.

Reply:
Discussed with @Andy-Millar. 'Save and continue' is the default govuk design system text for a save button.

I would be more inclined to name it `update` and alias it to `save`. But I thought that was overdoing it for now.

Reply:
Oh, sorry I was talking about the key not the content. Totally agree with `Save and continue` as the text. But the key is `save` which would suggest to me the content would be "Save", not "Save and continue". Just means if we ever need to introduce a "Save" action it won't clash.

Reply:
I was talking about the key too ;)

Reply:
Leave if it you're happy. I just felt `save` suggested the text "Save" not "Save and continue".

Reply:
Renamed ;)